modules/vendor/transform.lua

```diff
@@ -31,6 +46,10 @@ local transform = {}
 --     return state
 --   end
 
+-- Below an example of transforming uplink message from a device
+-- decoded values in string will be then stored in data_in
+payload.values.data_in = json.encode(transform.DecodeMode(payload.uplink))
```

Comment:
```suggestion
state.data_in = json.encode(transform.DecodeMode(state.uplink))
```
there is not payload argument here